### PR TITLE
Add support for non-pagination metadata.

### LINKED ISSUE #147: Unable to send meta data in GET response without including pagination attributes
Currently we are invoking a GET request to Django with some filter parameters applied. The results wont be more than 20 or 30, so we are not using pagination. 

The problem is we are unable to send meta data without the pagination attributes due to the following check in the `normalizeResponse` function

      if (!Ember.isNone(payload) &&
          payload.hasOwnProperty('next') &&
          payload.hasOwnProperty('previous') &&
          payload.hasOwnProperty('results')) {
          ...
      }

So we are sending dummy count, next and previous values and adding our meta data along with the results array. 
Could some one please suggest me a better way of doing this or to stick with the current approach?
Thank you.

### PR DESCRIPTION
**Pull request by benkonrath**
Fixes #147

### CODE REVIEW

**Comment by benkonrath** [comment]
@dustinfarris This is ready for review.

File: addon/serializers/drf.js [hunk 1]
```diff
@@ -103,10 +100,10 @@ export default DS.RESTSerializer.extend({
       convertedPayload['meta'] = JSON.parse(JSON.stringify(payload));
 
       // The next and previous pagination URLs are parsed to make it easier to paginate data in applications.
-      if (!Ember.isNone(convertedPayload.meta['next'])) {
+      if (convertedPayload.meta.hasOwnProperty('next') && !Ember.isNone(convertedPayload.meta['next'])) {
```
Comment (dustinfarris) [inline addon/serializers/drf.js]: I think this could be simplified as:

```js
if (!Ember.isNone(convertedPayload.meta.next) {
```

**Comment by benkonrath** [comment]
@dustinfarris Thanks, I made the change and pushed a new version (squashed into one commit).

**Comment by dustinfarris** [comment]
My concern with this is that it effectively makes `results` a reserved keyword.  I think we need to know more about the "additional metadata" use-case before we move to support it.

**Comment by benkonrath** [comment]
Good point. Let's wait to hear back from @indvinoth on #147. Perhaps the better option is that @indvinoth overrides `normalizeResponse` for his specific use case.

**Comment by benkonrath** [comment]
@dustinfarris After seeing the DRF code in #147, I agree that we should not merge this PR.  Overriding `normalizeResponse` is the best solution for the use case described in #147.

**Pull request closed without merging**